app/routes/account.rb

```diff
@@ -34,6 +34,30 @@ class Account < Core
         current_user.reset_key
         redirect "/account"
       end
+
+      post '/account/track-mentor/invite' do
```

Comment:
Oh yeah... now I totally see the routes redefined in `app.rb`. Ah well! Good to know haha.